src/main.rs

```diff
@@ -194,8 +194,10 @@ pub fn start(mut os_input: Box<dyn OsApi>, opts: Opt) {
                 let mut command_is_executing = command_is_executing.clone();
                 move || {
                     if let Some(layout) = maybe_layout {
+                        pty_bus.spawn_tab();
                         pty_bus.spawn_terminals_for_layout(layout);
                     } else {
+                        pty_bus.spawn_tab();
```

Comment:
I don't think we need this here. When we create a new `Screen` in the beginning of `start()` function, we create a new tab as well (inside `Screen::new()`). Spawning a new tab here would mean creating two tabs at startup.

Reply:
Hmm, I guess I forgot to clean that up, we aren't spawning a tab inside Screen::new() anymore actually